### PR TITLE
Doc/cli

### PR DESCRIPTION
**Pull request by lisonferez**
Add doc for command "import_vcard" in doc/for-developers and link to this doc, in README.rst in py.CLI

### CODE REVIEW

File: doc/for-developers/vcard_doc.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Import vcard
+
+We can import one or more vcards from a file.
```
Comment (gdchamal) [inline doc/for-developers/vcard_doc.md]: give context how this import is possible (using the caliopen cli)

File: doc/for-developers/vcard_doc.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Import vcard
+
+```
+cd src/backend
+caliopen import_vcard --help
+caliopen -f configs/caliopen.yaml.template import_vcard -u <username> [-d <directory> | -f <file>]
```
Comment (gdchamal) [inline doc/for-developers/vcard_doc.md]: shall give the equivalent command using cli in docker container (docker-compose run cli ....)

File: doc/for-developers/vcard_doc.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+**NB**: Only file with extension *.vcf* or *.vcard* are imported.
+
+## Working
+
+* src/backend/main.py/main.caliopen_main/parsers
+
+In vcard.py, we find functions `parse_vcard` and `parse_vcards` which read one or more vcards and return a NewContact or an array of NewContact. 
+
+* src/backend/tools/py.CLI/calioepn_cli/commands
```
Comment (iamdey) [inline doc/for-developers/vcard_doc.md]: `calioepn`

Comment (lisonferez) [inline doc/for-developers/vcard_doc.md]: Bien vu

**Pull request merged**